### ISSUE TITLE
[Bug?]: “yarn install --check-resolutions” is in the doc but isn't supported

### ISSUE THREAD

**Issue by alexislefebvre** [yarnpkg/berry#5813]
### Self-service

- [ ] I'd be willing to implement a fix

### Describe the bug

The documentation for `yarn install` mentions `--check-resolutions`: https://yarnpkg.com/cli/install#options-check-resolutions

But it doesn't work with yarn `3.6.4`.

This option was merged one year ago to `master`, yet it isn't available?

- https://github.com/yarnpkg/berry/pull/4302

### To reproduce

```
$ yarn -v
3.6.4
$ yarn install --check-resolutions
Unknown Syntax Error: Unsupported option name ("--check-resolutions").
```

One-liner to reproduce the issue:

```bash
docker run --rm node:16.20.2-alpine3.18 sh -c "yarn set version 3.6.2 && yarn --version && yarn install --check-resolutions"
```

### Environment

```shell
System:
  OS: Linux 5.15 Alpine Linux
  CPU: (16) x64 AMD Ryzen 7 5800X 8-Core Processor
Binaries:
  Node: 16.20.2 - /tmp/xfs-2f56aeb6/node
  Yarn: 3.6.4 - /tmp/xfs-2f56aeb6/yarn
  npm: 8.19.4 - /usr/local/bin/npm
npmPackages:
  jest: ^26.4.2 => 26.6.3
```

**Comment by arcanis** [comment]
It is available in RCs, soon in stable.

**Comment by alexislefebvre** [comment]
Thanks, it works with the RC:

```
docker run --rm node:18-alpine3.18 sh -c "yarn set version 4.0.0-rc.53 && yarn --version && yarn install --check-resolutions"
```

**Comment by alexislefebvre** [comment]
What about updating the doc to inform users that `--check-resolutions` is only available in the RC? I can't find how to update and contribute to the page https://yarnpkg.com/cli/install#options-check-resolutions

**Comment by alexislefebvre** [comment]
It looks like the source of the documentation is here: https://github.com/yarnpkg/berry/tree/master/packages/docusaurus

See also:

- https://github.com/yarnpkg/website/pull/1217

**Comment by arcanis** [comment]
This is fixed now that the 4.0.0 is released

**Comment by alexislefebvre** [comment]
Great news! Thanks.
